platform/engineering/frontend/platform-assignments.md

```diff
@@ -0,0 +1,40 @@
+# Overview 
+
+`vets-website` has several parts that require ongoing maintenance and improvement. These platform parts are assigned a primary and a secondary engineer. 
+
+This document describes the responsibilities of each of these roles and is intended for VSP frontend engineers and their leadership group. 
+
+## Primary 
+
+- Accountable for the ongoing maintenance of their designated platform part 
+  - Expected to do around a day of unplanned maintenance per sprint- this does not include support time but should be documented in a ticket and draft PR
+  - Inform engineering lead and product manager if no maintenance has been performed for 2 consecutive sprints 
+- Assists product manager with backlog management 
```

Comment:
Be ready to talk to a narrative or vision for current state and ideal state of your platform part
- How does this help your customer? 
- What could we do to better improve the experience? 
- What does the future look like to you? 
- Is context or feedback needed? (facilitate conversations with product so we can go after it)
- What metrics are\can we use to measure success?

Reply:
Updated. I don't think they should be responsible for a narrative or vision mostly because I want junior / mid level engineers to be able to own these parts.